doc/man3/SSL_CTX_set_options.pod

```diff
@@ -141,6 +141,11 @@ responsible for cleansing all other buffers. Most notably, this
 applies to buffers passed to functions like L<SSL_read(3)>,
```

Comment:
I would change the opening para to say something like "By default TLS and QUIC SSL objects keep a copy..."

Reply:
Done